src/fetch.ts

```diff
@@ -77,7 +77,9 @@ export function createFetch ({ fetch }: CreateFetchOptions): $Fetch {
         request = withQuery(request, opts.params)
       }
       const hasPayload = payloadMethods.includes((opts.method || '').toLowerCase())
-      if (opts.body && typeof opts.body === 'object' && hasPayload) {
+      const supportsFormData = typeof FormData === 'function'
+      const isFormData = supportsFormData && opts.body instanceof FormData
+      if (opts.body && typeof opts.body === 'object' && !isFormData && hasPayload) {
```

Comment:
how about something like this instead:
```suggestion
      if (opts.body && typeof opts.body === 'object' && hasPayload && opts.body.toString() !== '[object FormData]') {
```

Reply:
Pushed a generic fix but seems a good idea for using `[object FormData]` to append proper content-type (adding in short)